frontend/admin/src/pages/purchase/PurchaseDetail.tsx

```diff
@@ -416,7 +434,7 @@ const PurchaseDetailInternal = (props: any) => {
                               id="internalComments"/>
                 </div>
                 <div className="form-group">
-                    <label htmlFor="displayTableOrderItems">Einkaufsliste</label>
+                    <label htmlFor="displayTableOrderItems"><i>Die Einkaufsliste kann nicht mehr verändert oder gepeichert werden, nachdem der Helfer oder die Helferin den Einkauf abgeschlossen hat.</i></label>
```

Comment:
You should not be able to edit after accepting not after completion (there might be receipts already)

Reply:
Removed the input field and trash cans completely after VOLUNTEER_ACCEPTED